plugins/modules/network_settings_workflow_manager.py

```diff
@@ -1825,7 +1823,7 @@ def get_want_network(self, network_management_details):
                     clientAndEndpoint_aaa.get("protocol")
                 })
             else:
-                self.msg = "missing parameter protocol in clientAndEndpoint_aaa"
+                self.msg = "missing parameter protocol in client_and_endpoint_aaa"
```

Comment:
Can we write a message like the above?
Failed to process client_and_endpoint_aaa due to missing  parameter 'protocol'  in the playbook.